examples/kurth/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _examples-kurth:
```

Comment:
Awesome, add `examples/kurth/README.rst` in the table of contents in `docs/source/usage/examples.rst` please to include the file when it gets rendered to HTML/LaTeX/EPUB :)

Reply:
done via 13b09014294ffe1ca32ee8aab506369831bb0ed2